src/Indicator.vala

```diff
@@ -225,8 +230,8 @@ public class Notifications.Indicator : Wingpanel.Indicator {
     }
 
     private void update_tooltip () {
-        uint number_of_notifications = Session.get_instance ().count_notifications ();
```

Comment:
Can the `count_notifications` method be removed then? It doesn't seem to be used by anything else

Reply:
Done.